src/components/Pagination/docs/examples.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import { Pagination } from '../Pagination';
+
+export const importExample = `import { Pagination } from 'wix-ui-tpa/Pagination';`;
+
+export const example = `
+<Pagination 
+  maxPagesToShow={5}
+  totalPages={10}
+  currentPage={1}
+/>
+`;
+
+export const mobileExample = `
+<TPAComponentsProvider value={{ mobile: true }}>
+  <Pagination 
+    maxPagesToShow={5}
+    totalPages={10}
+    currentPage={1}
+  />
+</TPAComponentsProvider>
+`;
```

Comment:
We should probably add the `MockSettings` panel, as we have in other stories (see https://github.com/wix/wix-ui-tpa/blob/master/src/components/LikeButton/docs/extendedExamples.tsx for an example).
This will let us see that the style wiring works the way we want it to.